### PR TITLE
Address golint warning for package pkg

### PR DESCRIPTION
**Pull request by candysmurf**

### CODE REVIEW

File: pkg/aci/aci.go [hunk 1]
```diff
@@ -33,11 +33,17 @@ import (
 )
 
 var (
+	// ErrChmod - Error message for error changing file permission
```
Comment (tiffanyfj) [inline pkg/aci/aci.go]: Unless golint is specifically asking for it, you could just remove the error name - and start at Error message.

File: pkg/chrono/chrono.go [hunk 2]
```diff
@@ -31,7 +31,7 @@ type chrono struct {
 	pausedAt time.Time
 }
 
-// Now() should replace usage of time.Now(). If chrono has been paused, Now()
+// Now should replace usage of time.Now(). If chrono has been paused, Now()
```
Comment (tiffanyfj) [inline pkg/chrono/chrono.go]: I'd leave the () after the first Now.

File: pkg/schedule/simple_schedule.go [hunk 3]
```diff
@@ -27,7 +31,7 @@ func (s *SimpleSchedule) Validate() error {
 	return nil
 }
 
-// Waits until net interval and returns true. Returning false signals a Schedule is no
+// Wait until net interval and returns true. Returning false signals a Schedule is no
```
Comment (danielscottt) [inline pkg/schedule/simple_schedule.go]: I think this comment is outdated.

File: pkg/schedule/windowed_schedule_test.go [hunk 4]
```diff
@@ -29,7 +29,7 @@ func TestWindowedSchedule(t *testing.T) {
 			err := w.Validate()
 			So(err, ShouldBeNil)
 
-			r := make([]Response, 0)
+			var r []Response
```
Comment (danielscottt) [inline pkg/schedule/windowed_schedule_test.go]: +1000 on these!

Comment (tiffanyfj) [inline pkg/schedule/windowed_schedule_test.go]: Haha, yea I actually fixed one of these things a long time ago, just never touched the rest.

**Comment by danielscottt** [comment]
LGTM, @candysmurf!  Merging.

**Pull request merged**